showcase/app/styles/showcase-components/frame.scss

```diff
@@ -52,4 +52,11 @@ $shw-frame-navigation-bar-height: 48px;
   width: 100%;
   height: calc(100% - #{$shw-frame-navigation-bar-height});
   border: none;
+}
+
+// TESTING
+// Make frame wider so desktop view can be seen when included in a showcase page
+// TODO: Consider updating --main-width for all pages
+.showcase-app:has(.shw-frame) {
+  --main-width: calc(1088px + 2 * 2rem);
```

Comment:
The framed examples displayed within the AppFrame Showcase weren't wide enough to see the Desktop view. You could only see Mobile view. I wondered if we could modify the width of the main content to make it 1088px wide?

(NOTE: When/if the responsive layout behavior of the AppHeader is refactored this may resolve this issue although it could still affect the SideNav examples if they continue to use a fixed breakoint of 1088px.)

Reply:
No objections in principle. I have opened this PR https://github.com/hashicorp/design-system/pull/2312 to see what is the impact on the existing pages and the visual regressions.
If we are OK with it we can promote that to production, and you can remove this override here.

Reply:
I'm undoing the overrides, thank you.

Reply:
`main` has been merged in with the updated width for Showcase examples, thanks!